src/offline.js

```diff
@@ -100,49 +99,46 @@ export function startSync (cozy, doctype, timer) {
 export function hasSync (cozy, doctype) {
   return cozy._offline !== null &&
     doctype in cozy._offline &&
-    cozy._offline[doctype].autoSync !== null
+    !cozy._offline[doctype].autoSync
```

Comment:
Not sure that `aVar !== null` is equivalent to `!aVar`. *N.B. I know it is my code and my commit ;)*